src/app/globals.css

```diff
@@ -3,25 +3,20 @@
 @tailwind utilities;
 
 :root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
-}
+  --theme-color-primary: #057CDE;
+  --theme-color-secondary: #035496;
+  --theme-color-accent: #59B1F9;
+  --theme-color-text: white;
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    --foreground-rgb: 255, 255, 255;
-    --background-start-rgb: 0, 0, 0;
-    --background-end-rgb: 0, 0, 0;
-  }
+  --background-gradiant-light: #004278;
+  --background-gradiant-dark: #030304;
```

Comment:
[Nit]: Really not a big deal but did you mean to write `gradient` instead of `gradiant` for these?

Reply:
I definitely meant to write `gradient`, good catch!